tasks.py

```diff
@@ -422,6 +419,7 @@ def dev_env(ctx, architecture="amd64", name="kind", protocol=None, frr_volume_di
 # Configure MetalLB in the dev-env for layer2 testing.
 # Identify the unused network address range from kind network and used it in configmap.
 def layer2_dev_env():
+    _fetch_kubectl()
```

Comment:
no needed. Part of dev_env task

Reply:
I thought about it, but I prefer to be on the safe side and this is idempotent, so doesn't hurt.
This will make this function future proof in case we start using it from elsewhere.